spring-boot-admin-server-ui/src/main/frontend/views/instances/jolokia/m-bean-operation-invocation.vue

```diff
@@ -168,8 +168,8 @@
         this.state = 'executing';
         try {
           const result = await this.onExecute(this.args);
-          if (result.data.status < 400) {
-            this.result = result.data.value;
+          if (result.status < 400) {
```

Comment:
Does this still work for single instance? Or was was the code wrong before?

Reply:
The old code works only on single instances as the data field contains a list for multiple instances. The result object contains a status field with the same HTTP code.